flow/IThreadPoolTest.actor.cpp

```diff
@@ -11,7 +11,11 @@
 
 void forceLinkIThreadPoolTests() {}
 
+static ThreadReturnPromiseStream<std::string> notifications;
```

Comment:
Instead of using a static, could this be a pointer on `ThreadNameReceiver`? Personally, would prefer to just have a separate `IThreadPoolReceiver` rather than the extra control flow, but I don't feel strongly enough to insist.

Reply:
Done.